projects/cm_mcu/CommandLineTask.c

```diff
@@ -156,62 +156,88 @@ typedef struct __attribute__((packed)) {
 
 extern struct dev_i2c_addr_t pm_addrs_dcdc[];
 
+static BaseType_t reset_all(int argc, char **argv, char *m)
```

Comment:
Can you change this to a less generic name, maybe `snapshot_reset` or `sn_reset`?

Reply:
Done